README.md

```diff
@@ -35,31 +111,32 @@ a development environment.
 Options:
 
 * `-l`, `--language <LANGUAGE>`\
-  Which language to parse and format [possible values: json, toml]
+  Which language to parse and format [possible values: `json`, `toml`,
+  `ocaml`, `ocaml-implementation`, `ocaml-interface`].
```

Comment:
Is this part of the README generated?

Reply:
No. It probably should be... I'll add it as an issue.

Reply:
Done: Issue #277